File: 1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        vector<int> freq(26, 0);
        for(int i = 0; i<s.size(); i++) {
            freq[s[i] - 'a']++;
            freq[t[i] - 'a']--;
        }

        int diff = 0;
        for(int i = 0; i<26; i++) {
            diff += abs(freq[i]);
        }

        return diff/2;
    }
};

File: 1347-minimum-number-of-steps-to-make-two-strings-anagram/1347-minimum-number-of-steps-to-make-two-strings-anagram.java
class Solution {
    public int minSteps(String s, String t) {
        int[] freq = new int[26];
        for(int i = 0; i<s.length(); i++) {
            freq[s.charAt(i) - 'a']++;
            freq[t.charAt(i) - 'a']--;
        }

        int diff = 0;
        for(int i = 0; i<26; i++) {
            diff += Math.abs(freq[i]);
        }

        return diff/2;
    }
}